### DISCUSSION TITLE
[Feature]  text detection

### DISCUSSION THREAD

**Discussion by krzemyczek** [immich-app/immich#10399] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

detecting text on the photo, so that you can search for a photo by typing the phrase that is on the photo

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by krzemyczek** [comment]
and yes, i know it's complex thing, but i though it's worth mention, and btw, thank you for your excelent app :)

**Comment by bo0tzz** [comment]
#3168

**Reply by krzemyczek** [reply]
oh, i looked, but missed, thanks!
